src/scene/vcFlythrough.cpp

```diff
@@ -42,7 +44,7 @@ void vcFlythrough::AddToScene(vcState *pProgramState, vcRenderData * /*pRenderDa
     {
       if (m_exportInfo.currentFrame >= 0)
       {
-        if (vcTexture_SaveImage(pProgramState->screenshot.pImage, vcRender_GetSceneFramebuffer(pProgramState->pActiveViewport->pRenderContext), udTempStr("_temp/%05d.png", m_exportInfo.currentFrame)) != udR_Success)
+        if (vcTexture_SaveImage(pProgramState->screenshot.pImage, vcRender_GetSceneFramebuffer(pProgramState->pActiveViewport->pRenderContext), udTempStr("%s/%05d.png", m_exportPath[0] == '\0' ? "_temp" : m_exportPath, m_exportInfo.currentFrame)) != udR_Success)
```

Comment:
Don't default to `_temp`, just disable the export button until the user sets a location. Or, prompt for a location when the user clicks export, and then in the callback do what the button currently does.